src/search-manager/SearchSortWidget.tsx

```diff
@@ -62,13 +86,18 @@ export const SearchSortWidget: React.FC<Record<never, never>> = () => {
         size="sm"
       >
         <Icon src={SwapVert} className="d-inline" />
-        {searchSortLabel}
+        <div className="search-sort-toggle-label">{toggleLabel}</div>
```

Comment:
```suggestion
        <div className="py-0 px-1">{toggleLabel}</div>
```

You can achieve the same styling by using the [bootstrap classes](https://getbootstrap.com/docs/4.0/utilities/spacing/) without the need to add the new CSS file `SearchSortWidget.scss`

Reply:
Ach, thank you @yusuf-musleh ! Much cleaner this way.